src/hooks/API.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react';
+import { bobaList, toppingsList, milkList } from 'assets/sampleBobaAPI';
+
+const useApi = (url: string, endpoint: string) => {
+  // eslint-disable-next-line  @typescript-eslint/no-explicit-any
+  const [data, setData] = useState<any>();
+  const [error, setError] = useState<boolean>(false);
+  const [loading, setLoading] = useState<boolean>(true);
+  const fetchApi = () => {
+    const fullurl = url + endpoint;
+    if (process.env.NODE_ENV === 'development') {
+      fetch(fullurl)
+        .then((response) => {
+          return response.json();
+        })
+        .then((json) => {
+          setData(json);
+        })
+        .catch(() => {
+          setError(true);
+        })
+        .finally(() => {
+          setLoading(false);
+        });
+    } else {
+      if (endpoint == '/bobalist') {
+        setData(bobaList);
+      } else if (endpoint == '/toppings') {
```

Comment:
You are adding changes to the wrong branches
This should belong in toppings/milk branch

Reply:
Remove the `/toppings` and `/milk` for now

Reply:
b86981bc2aa11fecb29a1cdaa47b7c0988def1a8 updated